src/osx/cocoa/window.mm

```diff
@@ -2734,7 +2734,9 @@ void wxOSXCocoaClassAddWXMethods(Class c, wxOSXSkipOverrides skipFlags)
     if ( m_osxView )
         CFRetain(m_osxView);
     [m_osxView release];
-    m_osxView.clipsToBounds = YES;
+
+    if ( flags & Widget_IsUserPane )
```

Comment:
Very minor, but if you amend this to use `IsUserPane()` instead, please also do `s/they're/their/` in the commit message as this triggers my OCD :-/